src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +159,186 @@ export default async function() {
+
+For example, to have a local file work as an embedded replica to an encrypted libSQL server, you can set the following environment variables:
+
+```dotenv title=".env"
+ASTRO_DB_REMOTE_URL=file://local-copy.db?encryptionKey=your-encryption-key&syncInterval=60&syncUrl=libsql%3A%2F%2Fyour.server.io
+ASTRO_DB_APP_TOKEN=token-to-your-remote-url
+```
+
+:::caution
+
+Using a database file is an advanced feature, and care should be taken when deploying to prevent overriding your database and losing your production data.
+
+Additionally, this method will not work in serverless deployments, as the file system is not persisted in those environments.
+
+:::
+
+#### `encryptionKey`
+
+libSQL has native support for encrypted databases. Passing this search parameter will enable encryption using the given key:
+
+```dotenv title=".env"
+ASTRO_DB_REMOTE_URL=file:path/to/file.db?encryptionKey=your-encryption-key
+```
+
+#### `syncUrl`
+
+Embedded replicas are a feature of libSQL clients that enables a full synchronized copy of your database on a local file or in memory for ultra-fast reads. Writes are sent to a remote database defined on the `syncUrl` and synchronized with the local copy.
+
+Use this property to pass a separate connection URL to turn the DB into an embedded replica of another database. This should only be used with the schemes `file:` and `memory:`. The parameter must be URL encoded.
+
+For example, to have an in-memory embedded replica of a database on `libsql://your.server.io`, you can set the connection URL as such:
+
+```dotenv title=".env"
+ASTRO_DB_REMOTE_URL=memory:?syncUrl=libsql%3A%2F%2Fyour.server.io
+```
+
+#### `syncInterval`
+
+Interval between embedded replicas synchronizations in seconds. By default it only synchronizes on startup and after writes.
+
+This property is only used when `syncUrl` is also set. For example, to set an in-memory embedded replica to synchronize every minute set the following environment variable:
+
+```dotenv title=".env"
+ASTRO_DB_REMOTE_URL=memory:?syncUrl=libsql%3A%2F%2Fyour.server.io&syncInterval=60
+```
+
+### Self-hosted deployment
+
+Set the `ASTRO_DATABASE_FILE` environment variable to a path pointing to your `.db` file within the host environment during your build:
+
+```shell
+ASTRO_DATABASE_FILE=/srv/files/database.db astro build
+```
+
+The build will statically compile with this path as your production database. When you deploy and launch your server it will connect to the file at this path on the production host.
+
+Additionally, [pushing any table schema changes](#pushing-table-schemas) (also known as "schema migrations") must be managed manually using this environment variable.
+
+:::danger
+If you override your `.db` file on deployment, you will lose your production data. Follow the deployment method process for your host carefully to prevent data loss.
+:::
```

Comment:
We can remove this section now. The `ASTRO_DATABASE_FILE` is kept for backward compatibility until Studio is completely shut down. You can just use `ASTRO_DB_REMOTE_URL=file:path/to/database.db`.

Using the `file:` remote URL also allows the user to use push the schema using `astro db push` as normal instead of requiring manual changes.